src/main/java/com/exadel/movie_new/service/MovieServiceImpl.java

```diff
@@ -28,19 +36,29 @@ public Movie addMovie(Movie movie) {
     @Override
     public List<Movie> getAllMovies() {
         List<Movie> movieData = readJsonData();
+        if (movieData.isEmpty()) {

```

Comment:
In case you have some Collections entity - try to use double checking != bull and notEmpty, because it's two usual cases. I would recommend you use some utils like Collections.isEmpty(yourCollection) or any other. I think Spring boot must already contains something from the box. It could be any utill but the idea is it will always check for two possible cases !=null and notEmpty(). It will protest you from potential problems in future